ewcfg.py

```diff
@@ -3034,6 +3034,57 @@ def wef_scythe(ctn = None):
 		str_desc = "An unappetizing pile of Pawpaw Gruel. It’s just Pawpaw milled into something halfway between puke and diarrhea. The staple of a traditional Juvenile diet. ",
 		time_expir = milled_food_expir,
 	),
+	EwFood(
+		id_food = "khaotickilliflowerfuckenergy",
+		recover_hunger = 1200,
+		price = 120,
+		inebriation = 1000,
+		vendors = [vendor_mtndew, vendor_vendingmachine],
+		str_name = 'Khaotic Killiflower FUCK ENERGY',
+		str_eat = "You crack open a cold, refreshing can of Khaotic Killiflower flavored FUCK ENERGY. You throw your head back and begin to chug it, its viciously viscous consistency is almost enough to trigger your gag reflexes. But, you hold strong. Its bitter, low quality artificial Purple Killiflower flavorings remind you of discount children’s cough medicine. Nigh instanously, the chemicals infiltrate your central nervasystem. You feel an intense heat, like your body is about to spontaneously combust. You become lightheaded, your body twitching and convulsing randomly. And then, suddenly, you are launched into a manic, hyper-awareness. You begin to process more information in a single nanosecond than people with a masters in theoretical physics analyze in a lifetime. Your left and right brain sever, they now operate completely separately from one another and twice as efficiently. Your pineal gland doubles, nay, triples in size. You have never felt more alive. You crush the hand with can forehead, screaming.",
```

Comment:
instanously -> instantaneously
nervasystem -> nervous system
manic, hyper-awareness -> manic hyper-awareness
the hand with can forehead -> the can with your forehead